vm/src/compile.rs

```diff
@@ -16,7 +16,7 @@ struct Compiler {
     nxt_label: usize,
     source_path: Option<String>,
     current_source_location: ast::Location,
-    in_loop: bool,
+    loop_depth: usize,
```

Comment:
Did you see my comment [here](https://github.com/RustPython/RustPython/issues/497#issuecomment-466732095)? The `in_loop` flag should suffice.

Reply:
Changed.